.github/workflows/CI.yml

```diff
@@ -120,11 +123,13 @@ jobs:
       if: contains(matrix.os, 'ubuntu') || contains(matrix.os, 'macos')
       run: |
         ci/run_tests.sh
+        ci/run_tests.sh ${{ matrix.RELEASE_FLAGS }}
```

Comment:
We could separate the release build to a separate step to make it easier to inspect in the actions log.